shared/src/data/default-state/vehicle-templates.ts

```diff
@@ -84,35 +90,47 @@ const ktwKatSchutzVehicleTemplate = VehicleTemplate.create(
     `KTW (KatSchutz) ???`,
     ktwImage,
     2,
-    ['san', 'rettSan'],
-    ['standard']
+    [defaultPersonnelTemplates.san.id, defaultPersonnelTemplates.rettSan.id],
+    [defaultMaterialTemplates.standard.id]
 );
 
 const gwSanVehicleTemplate = VehicleTemplate.create(
     'GW-San',
     `GW-San ???`,
     gwSanImage,
     0,
-    ['gf', 'rettSan', 'rettSan', 'san', 'san', 'notarzt'],
-    ['big', 'big', 'big', 'big']
+    [
+        defaultPersonnelTemplates.gf.id,
+        defaultPersonnelTemplates.rettSan.id,
+        defaultPersonnelTemplates.rettSan.id,
+        defaultPersonnelTemplates.san.id,
+        defaultPersonnelTemplates.san.id,
+        defaultPersonnelTemplates.notarzt.id,
+    ],
+    [
+        defaultMaterialTemplates.big.id,
+        defaultMaterialTemplates.big.id,
+        defaultMaterialTemplates.big.id,
+        defaultMaterialTemplates.big.id,
+    ]
 );
 
 const nefVehicleTemplate = VehicleTemplate.create(
     'NEF',
     `NEF ???`,
     nefImage,
     0,
-    ['notarzt', 'notSan'],
-    ['standard']
+    [defaultPersonnelTemplates.notarzt.id, defaultPersonnelTemplates.notSan.id],
+    [defaultMaterialTemplates.standard.id]
 );
 
 const rthVehicleTemplate = VehicleTemplate.create(
     'RTH',
     `RTH ???`,
     rthImage,
     1,
-    ['notarzt', 'notSan'],
-    ['standard']
+    [defaultPersonnelTemplates.notarzt.id, defaultPersonnelTemplates.notSan.id],
+    [defaultMaterialTemplates.standard.id]
 );
 
 export const defaultVehicleTemplates: readonly VehicleTemplate[] = [
```

Comment:
```suggestion
const defaultVehicleTemplates: readonly VehicleTemplate[] = [
```

`defaultVehicleTemplates` is only used locally

Reply:
But it could be used externally and is allowed to be, so I will keep it this way.